regression/cbmc/object-bits-parsing/non_numeric.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+test.c
+--object-bits foobar
+object-bits must be positive and less than the pointer width
+^EXIT=1$
+^SIGNAL=0$
+--
+Test parsing of an invalid non numeric object-bits setting.
```

Comment:
This is a must-not-match, not a comment (similarly others)

Reply:
Fixed.